src/main/java/org/broadinstitute/hellbender/utils/pileup/PileupElement.java

```diff
@@ -387,7 +411,7 @@ public CigarElement getNextOnGenomeCigarElement() {
      * @return a CigarElement, or null if no such element exists
      */
     private CigarElement getNeighboringOnGenomeCigarElement(final Direction direction) {
```

Comment:
Document the fact that this method `getNeighboringOnGenomeCigarElement()` iterates until it finds an "on-genome" operator.